src/managers/GuildMemberManager.js

```diff
@@ -223,6 +223,19 @@ class GuildMemberManager extends CachedManager {
     return data.reduce((col, member) => col.set(member.user.id, this._add(member, cache)), new Collection());
   }
 
+  /**
+   * The data for editing a guild member.
+   * @typedef {Object} GuildMemberEditData
+   * @property {?string} [nick] The nickname to set for the member
+   * @property {Collection<Snowflake, Role>|RoleResolvable[]} [roles] The roles or role ids to apply
+   * @property {boolean} [mute] Whether or not the member should be muted
+   * @property {boolean} [deaf] Whether or not the member should be deafened
+   * @property {GuildVoiceChannelResolvable|null} [channel] Channel to move the member to
+   * (if they are connected to voice), or `null` if you want to disconnect them from voice
+   * @property {DateResolvable|null} [communicationDisabledUntil] The date or timestamp
```

Comment:
Why does this method not support TimeoutDateResolvable?

Reply:
Aren't DateResolvable and TimeoutDateResolvable the same thing now anyways?

Reply:
Documentation needs updating, but that makes total sense then, you can't disable communcation until a duration, but you can timeout for a duration

Reply:
IMO, the shortcut methods are a bridge between the client implementation and the API implementation, but then again, this isn't up to me or you. Its true, this is a little different from a large majority of the library, but I personally really like it.

Reply:
It's such an easy implementation that I don't see why such an inconsistency would be introduced, it makes a lot of sense imo

Reply:
Well then how about naming the property timeout in the edit method too? If it was done in the GuildMember method I don't see why not, right?

Reply:
Again, like I said earlier, IMO it is the *helpers* responsbility to bridge the gap, not the manager, the manager should remain fairly low level.

Reply:
Usually it's in the manager where all the processing of the data happens, not the helpers